src/test/java/org/folio/config/ConfigurationClientTest.java

```diff
@@ -0,0 +1,60 @@
+package org.folio.config;
+
+import static org.folio.config.ConfigurationsClient.MISSING_OKAPI_URL;
+import static org.folio.config.ConfigurationsClient.MISSING_TENANT;
+import static org.folio.config.ConfigurationsClient.MISSING_TOKEN;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+
+import org.folio.config.ConfigurationsClient.MissingHeaderException;
+import org.folio.rest.tools.client.HttpClientFactory;
+import org.folio.rest.tools.client.test.HttpClientMock2;
+import org.folio.util.model.OkapiHeaders;
+import org.junit.Test;
+
+import io.vertx.core.json.JsonObject;
+
+public class ConfigurationClientTest {
+
+  @Test
+  public void testVerifyOkapiHeaders() {
```

Comment:
The four test should go into 4 separate test methods. The first doesn't need a try-catch.
The others may use [assertThrows](https://github.com/junit-team/junit4/wiki/Exception-testing#using-assertthrows-method), but try-catch is also fine.